main.js

```diff
@@ -0,0 +1,194 @@
+// Opdracht 1 t/m 7. [Klikken op "SEARCH"-knop geeft output voor 1 t/m 7.]
+// Opdracht 1
+// Maak een 'Zoek'-knop op de pagina en koppel deze aan een functie die de gegevens over België ophaalt
+// en dit in de console logt. Tip: Als de de documentatie bekijkt en op async zoekt,
+// vindt je een voorbeeld van een GET-request.
+//
+// Opdracht 2.
+// Maak op basis van de response de volgende string en log dit in de console:
+// [country-naam] is situated in [subarea-name]. It has a population of [amount] people.
+//
+// Opdracht 3.
+// Maak op basis van de response de volgende string en log dit in de console: The capital is [city]
+//
+// Opdracht 4.
+// Maak een functie die ongeacht het aantal currencies die in een land gebruikt worden, een string maakt.
+// In een land kunnen één of twee currencies gebruikt worden:
+//  - 1 valuta: and you can pay with [currency]'s
+//  - 2 valuta's: and you can pay with [currency]'s and [currency]'s
+
+// 1.1 In HTML: already for next assignments: text field for text to search for in the API
+// 1.2 Add button in HTML script with type button en id="data-fetch-button". See HTML source code.
+
+// 1.2 Declare async function
+// 1.3 Try/catch block
+// 1.4 Make the request with axios + await
+
+// To check for other countries (assignment 5) use variable string:
+const countryName = 'belgium';
+
+// Opdracht 5.
+// Check of alles nog steeds werkt als je de gegevens over Aruba of Duitsland ophaalt!
+// Aruba: uncomment line 33 and comment line 28
+// const countryName = 'aruba';
+
+// Duitsland: uncomment line 36 and comment line 28
+// const countryName = 'germany';
+
+async function fetchDataBelgium() {
+    try {
+        const response = await axios.get(`https://restcountries.eu/rest/v2/name/${countryName}`);
```

Comment:
Wat ik zelf heel vervelend vind is dat ik heel veel `response.data[0]` moet herhalen. Wij developers zijn heel lui, dus is daarom de object spreader in het leven geroepen 😉 
```
const { name: countryName, population, subregion: sbuAreaName, ...otherProps } = response.data[0];
```

_Je hoeft de props niet perse te aliasen_